modules/visualization/dev-instance-foxbox/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+## Code of Conduct
```

Comment:
There is a CODE_OF_CONDUCT.md available in the root of ADDF repo. You can delete this here.